CODEOWNERS

```diff
@@ -0,0 +1,201 @@
+
+# E2E
+config/ @brunobar79 @skylarbarrera @osdnk 
+e2e/ @brunobar79 @ibrahimtaveras00 @skylarbarrera
+
+# IOS
+ios/ @brunobar79 @osdnk
+
+# SECURITY / APP INFRA
+patches/ @brunobar79 @osdnk
+scripts/ @brunobar79 @osdnk
+
+# i18N
+scripts/check-translations.sh @jinchung @marco
+scripts/codegen-translations.js @jinchung @marco
+
+# APOLLO
+src/apollo @brunobar79 @jinchung @derHowie
+
+# ASSETS
+src/assets @skylarbarrera @christianbaroni
+
+# COMPONENTS
+src/components @osdnk @terrysahaidak
+
+# APP CONFIG
+src/config @osdnk @brunobar79 @skylarbarrera
+
+# REACT CONTEXT
+src/context @osdnk @terrysahaidak
+
+# NETWORK DEBUGGING
+src/debugging @brunobar79
+
+# DESIGN SYSTEM
+src/design-system @markdalgleish @jxom
+
+# Typescript
+src/entities @jinchung @markdalgleish @marco
+
+# Handlers
+src/handlers @jinchung @brunobar79
+src/handlers/aesEncryption.ts @brunobar79 @estebanmino
+src/handlers/appEvents.ts @brunobar79 @derHowie
+src/handlers/assets.ts @jinchung @marco @derHowie
+src/handlers/authentication.ts @jinchung @brunobar79
+src/handlers/cloudBackup.ts @brunobar79 @skylarbarrera
+src/handlers/covalent.ts @brunobar79 @derHowie
+src/handlers/deeplinks.ts @brunobar79 @tchayen
+src/handlers/dispersion.ts @brunobar79 @derHowie @jinchung
+src/handlers/ens.ts @estebanmino @skylarbarrera @jxom
+src/handlers/gasFees.ts @estebanmino @brunobar79 @jinchung
+src/handlers/imgix.ts @brunobar79 @osdnk @skylarbarrera
+src/handlers/opensea-api.ts @jinchung @skylarbarrera
+src/handlers/poap.ts @skylarbarrera @osdnk
+src/handlers/svgs.ts @skylarbarrera @osdnk
+src/handlers/tokenSearch.ts @brunobar79 @jinchung @derHowie
+src/handlers/transactions.ts @brunobar79 @osdnk @jinchung
+src/handlers/uniswap.ts @brunobar79 @jinchung
+src/handlers/walletReadyEvents.ts @brunobar79 @jinchung
+src/handlers/web3.ts @brunobar79 @jinchung @estebanmino
+src/handlers/wyre.ts @jinchung @skylarbarrera
+
+# Helpers
+src/helpers @brunobar79 @jinchung
+
+
+# Hooks
+src/hooks @jinchung @osdnk 
+src/hooks/useENSProfile.ts @estebanmino @jxom
+src/hooks/useENSProfileForm.ts @estebanmino @jxom
+src/hooks/useENSRegistration.ts @estebanmino @jxom
+src/hooks/useENSRegistrationCosts.ts @estebanmino @jxom
+src/hooks/useGas.ts @estebanmino @brunobar79 @jinchung
+
+# Languages
+src/languages @skylarbarrera @marco
+
+
+# Model 
+src/model @brunobar79 @jinchung
+src/model/mmkv @osdnk @terrysahaidak
+
+# Navigation 
+src/navigation @osdnk @terrysahaidak
+
+# Parsers 
+src/parsers @brunobar79 @jinchung
+
+# rainbow fetch 
+src/rainbow-fetch @brunobar79 @skylarbarrera
+
+# raps
+src/raps @brunobar79 @jinchung
+src/raps/registerENS.ts @estebanmino @jxom
+
+# charts, shadow stack, cool modals, sheets
+src/react-native-animated-charts @osdnk @terrysahaidak
+src/react-native-cool-modals @osdnk @terrysahaidak
+src/react-native-shadow-stack @osdnk @terrysahaidak
+src/react-native-yet-another-bottom-sheet @osdnk @terrysahaidak
+
+# react query
+src/react-query @jinchung @derHowie
```

Comment:
am happy to be added here (from a react-query API perspective) as I did create this folder 😛